### ISSUE TITLE
Exposing API variable naming - confusing

### ISSUE THREAD

**Issue by gscallon** [defenseunicorns/leapfrogai#734]
### Environment
Device and OS: Ubuntu 
App/package versions: uds and all other dependencies up-to-date
Kubernetes distro being used: k3d
Other:

### Steps to reproduce
1. Attempt to expose the API in the LeapfrogAI UDS bundle
2. Put down `expose_openai_schema:true`
3. The API exposure does not work
4. Realize it is actually `expose_openapi_schema:true`
5. The API is exposed and works properly

### Expected result

We have had 3 different users thus far mess up the copy/paste or accidentally use `openai` in place of `openapi`. 

Normally, I would attribute this to user error, but three different users making the same mistake represents a pattern.

### Actual Result

The API is not exposed properly when the wrong variable is used.

### Severity/Priority

Nice-to-have. May be rendered obsolete if the API is exposed by default once the long-lived API key branch is merged into the project.

### Additional Context
CC @unicorn-kp

**Comment by gphorvath** [comment]
in #561 proposing to rename to `EXPOSE_API`
